### ISSUE TITLE
No support for NPM package previews

### ISSUE THREAD

**Issue by ergo** [webcomponents/webcomponents.org#1166]
Hello,

When using https://www.webcomponents.org/preview, there are still checks for bower.json presence so packages that are served from npm cannot be previewed.

Resulting error: 

> Error - 12
> Unable to process element
> Missing bower.json
> Currently, only Bower is supported. In the latest tagged version of this repository, ensure there is a valid bower.json.

**Comment by gkjohnson** [comment]
I ran into this today. Is this just for the preview? Are we able to publish without a `bower.json` file?

**Comment by ergo** [comment]
@gkjohnson Yes https://www.webcomponents.org/element/polymer-ui-router - I managed to publish an element here from NPM, just struggle with demos working.

**Comment by gkjohnson** [comment]
Interesting -- I'm seeing this error in the demos:

```
Uncaught TypeError: Failed to resolve module specifier "@webcomponents/shadycss/entrypoints/apply-shim.js". Relative references must start with either "/", "./", or "../".
```

Looks like the site doesn't resolve the bare module imports. I suppose a pre-built version of the element might be the best way to show a demo?

**Comment by ergo** [comment]
@samuelli is this on the roadmap somewhere?
